Sources/ForageSDK/Foundation/Network/Provider.swift

```diff
@@ -146,17 +146,30 @@ class Provider {
 
     private func processData<T: Decodable>(model: T.Type, data: Data?, response: HTTPURLResponse?, error: Error?, completion: @escaping (Result<T, Error>) -> Void) {
         guard let data = data else {
-            return completion(.failure(ForageError(errors: [ForageErrorObj(httpStatusCode: response?.statusCode ?? 500, code: "invalid_data", message: "Invalid Data")])))
+            return completion(.failure(ForageError.create(
+                code: "invalid_input_data",
+                httpStatusCode: response?.statusCode ?? 500,
+                message: "Double check the reference documentation to validate the request body, and scan your implementation for any other errors."
+            )))
```

Comment:
Updated to [more accurate/documented error code](https://docs.joinforage.app/reference/errors#invalid_input_data)